Frontend/frontend/src/App.js

```diff
@@ -19,6 +19,8 @@ import APIRequest from './Util/APIRequest';
 import NewFlat from './Components/NewFlat';
 import MembersPage from './Components/Members';
 import ForgotPassword from './Views/ForgotPassword';
+import ResetPassword from './Views/ResetPassword';
+import MembersPage from './Components/Members';
```

Comment:
Yo, this doesn't build for me because MembersPage is duplicated.

Reply:
Thank you for pointing this out. Have removed duplicate and should now render correctly :)